tests/request/test_request.py

```diff
@@ -130,15 +131,13 @@ def test_slot_behaviour(self):
             assert getattr(inst, at, "err") != "err", f"got extra slot '{at}'"
         assert len(mro_slots(inst)) == len(set(mro_slots(inst))), "duplicate slot"
 
-    async def test_context_manager(self, monkeypatch):
+    async def test_context_manager(self, monkeypatch, httpx_request):
```

Comment:
these changes look wrong to me. we monkeypath init & shutdown before using `httpx_request` as context manager - I don't think that we should use an already initialized object for that.

Reply:
yeah I just realized it yesterday. Guess I'll just have to use `NonchalantHTTPXRequest` for that line.